webpack.config.js

```diff
@@ -49,7 +49,38 @@ module.exports = function (_, env) {
       }
     },
     module: {
+      // Disable the default JavaScript handling:
+      defaultRules: [],
       rules: [
+        {
+          oneOf: [
```

Comment:
@developit Love your spelling of one-off here. Making clear it's really just a one-of.

Reply:
webpack gets the credit for this one, I'm just writing to their API :P